.github/workflows/lint_build.yaml

```diff
@@ -0,0 +1,90 @@
+name: Lint and Build
+
+on:
+  pull_request:
+
+jobs:
+  javascript:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10
+
+      - name: JavaScript - kyber
+        run: |
+          cd external/js/kyber
+          npm ci
+          npm run linter
+          npm run build
+
+      - name: JavaScript - cothority
+        run: |
+          cd external/js/cothority
+          npm ci
+          npm run linter
+          npm link @dedis/kyber
+          npm run build
+
+  golang:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        golang-version: [1.14, 1.15]
```

Comment:
Same suggestion regarding versions as environment variables.

Reply:
Is there a way to calculate the previous version? Or do I need to give `GO_VERSION` and `GO_VERSION_PREVIOUS`?

Reply:
One way I found to execute arbitrary code is using https://github.com/actions/github-script.
I used it for example [here](https://github.com/c4dt/dp3t-sdk-android/blob/microg-nearby/.github/workflows/nogaen.yml#L22-L31).

Reply:
Seems overly complicated to remove one line in the `env` section...

Reply:
That's up to you. :)